Samples/LineGraph/README.md

```diff
@@ -10,18 +10,25 @@ In order to use the library, you have to install it. This is done by running the
 uno install Fuse.Charting
 ```
 
-With that done, you should be all set.
+With that done, you should be all set. Remember to include `Fuse.Charting` package in your unoproj file, like it is done in this example:
+
+```
+"Packages": [
+    [...]
+    "Fuse.Charting"
+  ],
+```
 
 ## The line-graph renderer
 
-The line-graph renderer should function as a `ux:Class` we can place inside a `Plot` element to have a graph drawn the way we want it. We are therefore interrested in adding a few properties so we get more control over the graph:
+The line-graph renderer should function as a `ux:Class` we can place inside a `Plot` element, acting as a shortcut to a styled line graph. We are therefore interested in adding a few properties so we get more control over the graph:
 
 	<Panel ux:Class="MyPlot">
 		<int ux:Property="SeriesIndex" />
 		<float4 ux:Property="MainColor" />
 		<float4 ux:Property="BackgroundColor" />
 
-One property of interest, is `SeriesIndex`. If you provide multiple `DataSeries` to a `Plot`, you can use the `SeriesIndex` on a `PlotData` element to select which data series you are drawing from. By exposing the `SeriesIndex` as a property, followed by reading from it locally using `ReadProperty`(see below), we can control what data series our renderer is reading from.
+One property of interest, is `SeriesIndex`. If you provide multiple `DataSeries` to a `Plot`, you can use the `SeriesIndex` on a `PlotData` element to select which data series you are drawing from. By exposing the `SeriesIndex` as a property, followed by binding to it using `ReadProperty`(see below), we can control what data series our renderer is reading from.
```

Comment:
"followed by binding to" sounds weird. I suggest something like: "By exposing SeriesIndex` as a property, we can bind to it using a `ReadProperty`-binding. This lets us control what data......."